@here/harp-omv-datasource/lib/OmvDecoder.ts

```diff
@@ -167,14 +169,29 @@ export class OmvDecoder implements IGeometryProcessor {
         private readonly m_skipShortLabels = true,
         private readonly m_storageLevelOffset = 0,
         private readonly m_enableElevationOverlay = false,
-        private readonly m_languages?: string[]
+        private readonly m_languages?: string[],
+        politicalView?: string
     ) {
         const styleSetDataFilter = new StyleSetDataFilter(m_styleSetEvaluator);
         const dataPreFilter = m_dataFilter
             ? new ComposedDataFilter([styleSetDataFilter, m_dataFilter])
             : styleSetDataFilter;
+        // Create feature environment processor, it may be re-configured on decoder configure call
+        const localEnvProcessor = new OmvLocalEnvironmentProcessor();
+        // TODO: Later on it is possible to localize features names using the Env interface.
+        //if (m_languages !== undefined && m_languages!.length > 0) {
+        //    localEnvProcessor.setLocale("name", m_languages[0]);
+        //}
+        // Setup localization (point of view) for kind attribute
+        if (politicalView !== undefined && politicalView.length > 0) {
+            localEnvProcessor.setLocale("kind", politicalView);
+        }
+        this.m_envProcessor = localEnvProcessor;
```

Comment:
Please verify that the performance is not degrading (measure, please). In case there is a significant change, please make sure that
1. the localeEnvProcessor is only used for lines (Ask Stephan Kosmella)
2. the localEnvProcessor is only used if a politicalView is set, such that the performance is not degrading in the default case. In the default case the MapEnv may be used.